charts/nri-metadata-injection/values.yaml

```diff
@@ -17,6 +18,7 @@ image:
     # - name: regsecret
 
 jobImage:
+  registry:
   repository: k8s.gcr.io/ingress-nginx/kube-webhook-certgen
```

Comment:
Having the `registry` empty, in case they use the common library to set a global `registry` then will finish with something like `registry.tld/k8s.gcr.io/ingress-nginx/kube-webhook-certgen`

I am seeing issues with this approach but I don't know how to tackle them yet... And is the approach [we did in the metrics adapter](https://github.com/newrelic/newrelic-k8s-metrics-adapter/pull/43/commits/5e29d743bb2df9b22fae9b22761adcf5629f4f79).

Reply:
Done! [555c5f8](https://github.com/newrelic/k8s-metadata-injection/pull/163/commits/555c5f86dd06417a6b3e446e37e0f9d02c90b466)